src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import "variables";
+@import "reset";
+@import "normalaize";
```

Comment:
There is a typo in the import statement. It should be 'normalize' instead of 'normalaize'.